app/views/users/show.html.erb

```diff
@@ -1,2 +1,34 @@
 <h1> This is users page <%= @id %> </h1>
 
+<div class="wrapper-1">
+    <li class="item">  
+       <img class="photo" src="<%= @user.photo%>" alt="user photo">
+       <div class= "container"> 
+          <div class="name"> <%= @user.name %> </div>
+          <div class="post-num"> number of posts: <%= @user.posts_counter %></div>
+       </div>
+    </li>
+    <div class="container-column-2"> 
+      <div class="bio"> Bio </div>
+       <%= @user.bio %>
+    </div>
+    
+    <% @posts.each do |post| %>
+        <div class= "container-column-2"> 
+            <div class="item">Post # <%= post.id %>  <%= post.title %> </div>
+            <div class="item"> <%= post.text %> </div>
+
+            <div class="container-2">               
+              <div>Comments: <%= post.comments_counter %> </div>
+              <div>likes: <%= post.likes_counter %> </div>
+            </div>
+        </div>
+    <% end %>
+
+
+    
+
+    <button type="button" class="button-1">See all posts</button>
+  
```

Comment:
- [ ] It is required that `See all posts` link opens user's post index as displayed on this below wireframe.

### Wireframe
![wireframe](https://github.com/edahigure/blog-app-2/assets/26597432/699684b3-df36-4863-abd8-2708dc59a505)

### App Behaviour
[see-all-post.webm](https://github.com/edahigure/blog-app-2/assets/26597432/e0119c9a-9104-4ead-9cb9-e34d2edd7f69)